CHANGELOG.md

```diff
@@ -26,9 +26,6 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 - Adds `thrift.Named` option for appropriately labelling procedures inherited
   from other thrift services.
-- The HTTP protocol now marks peers as unavailable immediately when the remote
-side closes the connection.
```

Comment:
The changelog shouldn't be reverted. We need a new entry in the Unreleased section marking this change as reverted.